src/Components/Main/Pages/ExamplesPage/ExampleChart/ExampleChart.tsx

```diff
@@ -12,17 +13,20 @@ type PropsType = {
   addressError: Nullable<ErrorType>;
   options: IChart;
   setNewOptions: (exampleOptions: IChart) => void;
+  isInfoPage?: boolean;
+  tableValues?: (string | number)[][];
 };
 
 export const ExampleChart = memo<PropsType>(
-  ({ isFetchingAddress, address, addressError, options, setNewOptions }): ReactElement => {
+  ({ isFetchingAddress, address, addressError, options, setNewOptions, isInfoPage, tableValues }): ReactElement => {
     return (
       <div className={classes.exampleChart}>
         <div className={classes.wrapper}>
           <div className={classes.imageContainer}>
             <Chart isFetchingAddress={isFetchingAddress} address={address} addressError={addressError} />
           </div>
         </div>
+        {isInfoPage && tableValues && <Table tableValues={tableValues} />}
```

Comment:
The same here, isInfoPage is not the best naming here, you could try - "showTable", "hideTable" or smth else

Reply:
Done!